app/amp_start.py

```diff
@@ -13,25 +13,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import urlparse
 import start
+import urlparse
 import utils
 
 
 class Handler(start.Handler):
```

Comment:
Can you add a comment to explain what is the page served by this handler? (start.Handler also lacks such comment, but it would be useful to have, at least here.)

Reply:
done.